backend/e4h-services/ingestion-service/app/utils/icc_report_converter.py

```
@@ -0,0 +1,516 @@
+    for item in items:
+        k = key_fn(item)
+        if runs and runs[-1][0] == k:
+            runs[-1][1].append(item)
+        else:
+            runs.append((k, [item]))
+    return runs
+
+
+def zip_section(template_entries, excel_instances, section, unmatched):
+    output = {}
+    template_key_fn = lambda e: TEMPLATE_LABEL_MERGE.get(normalize_label(e[0]), normalize_label(e[0]))
```

Ruff (0.15.20)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `template_key_fn` as a `def`

(E731)